packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -80,27 +90,35 @@ class Carousel extends Foundation<
     public render(): React.ReactElement<HTMLDivElement> {
         return (
             <div {...this.unhandledProps()} className={this.generateClassNames()}>
-                <Flipper
-                    direction={FlipperDirection.previous}
-                    onClick={this.handlePreviousClick}
-                    className={get(
-                        this.props,
-                        "managedClasses.carousel_flipperPrevious",
-                        ""
-                    )}
-                />
+                {this.isSingleSlide && (
```

Comment:
we typically will not do any inline functionality within a rendering template like this, I would prefer to see a this.renderSingleSlide() method

Reply:
Changed